wdl/AnnotateFunctionalConsequences.wdl

```diff
@@ -0,0 +1,95 @@
+version 1.0
+
+import "Structs.wdl"
+
+workflow AnnotateFunctionalConsequences {
+  input {
+    File vcf
+    File vcf_index
+    String prefix
+
+    File protein_coding_gtf
+    File? noncoding_bed
+    Int? promoter_window
+    Int? max_breakend_as_cnv_length
+
+    String gatk_docker
+    RuntimeAttr? runtime_attr_svannotate
+  }
+
+  call SVAnnotate {
+    input:
+      vcf = vcf,
+      vcf_index = vcf_index,
+      prefix = prefix,
+      protein_coding_gtf = protein_coding_gtf,
+      noncoding_bed = noncoding_bed,
+      promoter_window = promoter_window,
+      max_breakend_as_cnv_length = max_breakend_as_cnv_length,
+      gatk_docker = gatk_docker,
+      runtime_attr_override = runtime_attr_svannotate
+  }
+
+  output {
+    File annotated_vcf = SVAnnotate.annotated_vcf
+    File annotated_vcf_index = SVAnnotate.annotated_vcf_index
+  }
+}
+
+task SVAnnotate {
+  input {
+    File vcf
+    File vcf_index
+    String prefix
+
+    File protein_coding_gtf
+    File? noncoding_bed
+    Int? promoter_window
+    Int? max_breakend_as_cnv_length
+
+    String gatk_docker
+    RuntimeAttr? runtime_attr_override
+  }
+
+  RuntimeAttr default_attr = object {
+    cpu_cores: 1,
+    mem_gb: 3.75,
+    disk_gb: 10,
+    boot_disk_gb: 10,
+    preemptible_tries: 3,
+    max_retries: 1
+  }
+  RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+  Float mem_gb = select_first([runtime_attr.mem_gb, default_attr.mem_gb])
+  Int java_mem_mb = ceil(mem_gb * 1000 * 0.7)
+
+  String outfile = "~{prefix}.annotated.vcf.gz"
+
+  output {
+    File annotated_vcf = "~{outfile}"
+    File annotated_vcf_index = "~{outfile}.tbi"
+  }
+  command <<<
+
+     set -euo pipefail
+
+     gatk --java-options "-Xmx~{java_mem_mb}m" SVAnnotate \
```

Comment:
What do you think about adding an `String additional_args` input that would let the user specify arbitrary arguments not built into the wdl (e.g. a requester pays project id)?